MediaGalleryUi/Test/Mftf/ActionGroup/AdminMediaGalleryApplySelectFilterActionGroup.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminMediaGalleryApplySelectFilterActionGroup">
+        <annotations>
+            <description>Applies select filter to the media gallery grid</description>
+        </annotations>
+        <arguments>
+            <argument name="selectLabel" type="string"/>
+            <argument name="selectOption" type="string"/>
```

Comment:
`filterLabel` and `optionLabel` argument names would be more descriptive for a developer